templates/techreport/category.html

```diff
@@ -0,0 +1,117 @@
+              {{ category_selected }}
+            </option>
+          </select>
+        </div>
+      </div>
+
+    </fieldset>
+    <fieldset class="lens">
+      <legend class="sr-only">Lens</legend>
+
+      <label for="geo">Geo</label>
+      <div class="select-label">
+        <select name="geo" id="geo">
+          <option value="ALL" selected="true">
+            ALL
+          </option>
+        </select>
+      </div>
+
+      <label for="rank">Rank</label>
+      <div class="select-label">
+        <select name="rank" id="rank">
+          <option value="ALL" selected="true">
+            ALL
+          </option>
+        </select>
+      </div>
+    </fieldset>
+  </div>
+  <button type="submit" id="submit-form">Update</button>
+  {% include "techreport/components/filter_meta.html" %}
+  <fieldset class="breakdown">
+    <legend>Breakdown</legend>
+    <div class="client-breakdown-wrapper">
+      <label for="client-breakdown">Client</label>
+      <select name="client-breakdown" id="client-breakdown">
+        {% if request.args.get('client', '') == 'desktop' %}
+        <option value="mobile">Mobile</option>
+        <option value="desktop" selected>Desktop</option>
+        {% else %}
+        <option value="mobile" selected>Mobile</option>
+        <option value="desktop">Desktop</option>
+        {% endif %}
+      </select>
+    </div>
+  </fieldset>
+</form>
+
+<template id="filter-option">
+  <option value="Option">
+    Option
+  </option>
+</template>
+
+<template id="tech-selector">
+  {% set technology = "ALL" %}
+  {% set name = "tech-new" %}
+  {% set key = 2 %}
+  {% include "techreport/templates/selector_tech.html" %}
+</template>
+{% endblock %}
+
+{% block section %}
+  {{ super() }}
+  <div class="report-content">
+    <div class="info-panel">
+      <div class="block-l">
+        <p class="info-label">Beta</p>
+        <p>We're still actively working on the categories page.</p>
+      </div>
+    </div>
+
+    <div class="intro block-m">
+      <div class="info">
+        <h1>
+          <span class="subtitle">Category</span>
+          <span class="">{{ tech_report_page.filters.category or 'CMS' }}</span>
+        </h1>
+      </div>
+    </div>
+
+    <div
+      id="comparison-summary"
+      class="card block-m summary report-section"
+      data-type="section"
+      data-api="cwv,lighthouse,page-weight"
+    >
+      <h2>Summary</h2>
+      {% set component = tech_report_page.config.tech_comparison_summary %}
+      {% set id = component.id %}
+      {% set client = request.args.get('client', '')  or 'mobile' %}
+      {% set sort_endpoint = "adoption" %}
+      {% set sort_metric = "adoption" %}
+      {% set sort_key = "origins" %}
+      {% set sort_order = "desc" %}
+
+      {% set table = component.table %}
+      {% include "techreport/components/table_linked.html" %}
+
+      <a data-name="selected-apps" href="/reports/techreport/tech" class="cta-link">Compare technologies</a>
```

Comment:
If there are geo/rank/client filters set on the Categories page and I click this to compare multiple technologies, it looks like those settings are lost on the comparison page.

USA, Top 100k, Desktop:

<img width="607" alt="image" src="https://github.com/user-attachments/assets/7f36769d-2d4d-4987-ba75-8ceb48d0fa1d" />

All, All, mobile:

<img width="1210" alt="image" src="https://github.com/user-attachments/assets/7eb078c1-0dac-47ec-a695-f32d0cf5ba3c" />